lambdatest/publish_script.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+MYVAR="deploy"
+zip from_travis_$MYVAR.zip lambda_function.py
```

Comment:
what's this? Also what is the program.prof? 😅

Reply:
🤦‍♀